docs/application/vstools/install.md

```diff
@@ -0,0 +1,170 @@
+
+- Intel&reg; Hardware Acceleration Execution Manager (Intel&reg; HAXM) speeds up the Tizen emulation on Intel-VT-enabled systems. The Intel&reg; HAXM installation is started automatically as part of the Visual Studio Tools for Tizen installation. For more information, see [Hardware Accelerated Execution Manager](../tizen-studio/setup/hardware-accelerated-execution-manager.md)
+
+- Make sure **Hyper-V** is disabled (in Windows&reg; 10 or higher):
+  1. In the **Start** menu, select **Programs and Features**.
+
+   ![Programs and Features](media/cs_prerequisite01-250x401.png)
+
+  2. Select **Turn Windows features on or off**.
+  3. Disable **Hyper-V** and click **OK**.
+
+   ![Disable Hyper-V](media/cs_prerequisite-disable-hiperv.png)
+
+  4. Reboot the computer.
+
+
+## Visual Studio Tools for Tizen Installation {#install}
+
+To use the Tizen SDK tools, you must install the Visual Studio Tools for Tizen extension and the baseline SDK.
+
+> **Note**  
+> If the Tizen Studio is already installed in your computer, simply [set the Tizen baseline SDK path](#set-tizen-baseline-sdk) to the existing SDK instead of reinstalling it.
+
+
+### Installing the Extension
+
+The Visual Studio Tools for Tizen extension is registered in the Visual Studio Marketplace. You can install extensions from the Visual Studio Marketplace in the Visual Studio IDE.
+
+1. In the Visual Studio IDE menu, go to **Tools &gt; Extensions and Updates**.
+2. In the Visual Studio Marketplace, search for **Tizen**.
+
+   ![Marketplace](media/howtoinstall-extensionwindows.png)
+
+3. Click **Download** and close the Visual Studio IDE.
+
+   The installation starts.
+
+Alternatively, you can download the Visual Studio Tools for Tizen extension from the [Visual Studio Marketplace Web site](https://marketplace.visualstudio.com/items?itemName=vs-publisher-1484655.VisualStudioToolsforTizen).
+
+### Installing the Tizen Baseline SDK
+
+After installing the Visual Studio Tools for Tizen extension, you must set up the Tizen baseline SDK:
+
+1. In the Visual Studio IDE menu, go to **Tools &gt; Tizen &gt; Tizen Package Manager**.
+2. Select **Install new Tizen SDK**.
+
+   ![Select new installation](media/howtoinstall-installwizard1.png)
+
+3. Read the license document and click **I Agree**.
+
+   ![Agree to license details](media/howtoinstall-installwizard2.png)
+
+4. Enter the root directory path where you want to install and click **Next**.
+
+   ![Set the installation path](media/howtoinstall-installwizard3.png)
+
+   The Tizen SDK installer is downloaded and the baseline SDK is installed automatically.
+
+   ![Installer download](media/howtoinstall-installwizard4.png)
+
+   ![Baseline SDK installation](media/howtoinstall-installwizard5.png)
+
+   Finally, the Tizen Package Manager installs the Tizen SDK tools.
+
+   ![Tool installation](media/howtoinstall-installwizard6.png)
+
+### Setting the Tizen Baseline SDK Path {#set-tizen-baseline-sdk}
```

Comment:
It looks like it should be in form `<a name="set-tizen-baseline-sdk"></a>`.

Reply:
done